piqe_ocp_lib/api/resources/ocp_cluster_versions.py

```diff
@@ -73,3 +74,103 @@ def get_cluster_id(self):
         logger.info("Cluster ID : %s", cluster_id)
 
         return cluster_id
+
+    def _build_spec(self, cv_body: Dict) -> Dict:
+        """
+        """
+        cluster_version = self.get_cluster_version().items[0]
+        cv_body.update({"apiVersion": self.api_version, "kind": self.kind})
+        metadata = {
+            "name": "version",  # hardcoded value in ClusterOperator's API
+            "resourceVersion": cluster_version.metadata.resourceVersion,  # Required for successfull API call
+        }
+        spec = {"clusterId": cluster_version.spec.clusterID}  # Required for successfull API call
+
+        cv_body["metadata"] = {**cv_body.get("metadata", {}), **metadata}
+        cv_body["spec"] = {**cv_body.get("spec", {}), **spec}
+        return cv_body
+
+    def available_updates(self) -> List[str]:
+        """
+        Get available versions to update
+        :return: list of available versions
+        """
+        cluster_version = self.get_cluster_version()
+        available_updates = cluster_version.items[0].status.availableUpdates
+
+        try:
+            return [element.version for element in available_updates[::-1]]
+        except TypeError:
+            logger.debug("There's no update available.")
+
+        return []
+
+    def available_channels(self) -> List[str]:
```

Comment:
We can make this a more rich api by allowing the user the ability to filter by type of channel, example: 'candidate, fast, stable'. So maybe make the method like 'def available_channels(self, type=' '):